server/client.go

```diff
@@ -198,13 +198,14 @@ type outbound struct {
 	s   []byte        // Secondary for use post flush
 	nb  net.Buffers   // net.Buffers for writev IO
 	sz  int           // limit size per []byte, uses variable BufSize constants, start, min, max.
-	sws int           // Number of short writes, used for dyanmic resizing.
+	sws int           // Number of short writes, used for dynamic resizing.
 	pb  int64         // Total pending/queued bytes.
 	pm  int64         // Total pending/queued messages.
 	sg  *sync.Cond    // Flusher conditional for signaling.
-	fsp int           // Flush signals that are pending from readLoop's pcd.
-	mp  int64         // snapshot of max pending.
 	wdl time.Duration // Snapshot fo write deadline.
+	mp  int64         // snapshot of max pending.
+	fsp int           // Flush signals that are pending from readLoop's pcd.
+	lws int64         // Last write size
```

Comment:
Not sure if this was during dev because I see it being set but not used anywhere?

Reply:
I was using it to determine releaseLock state but got more consistent results with what is there. I can remove.